pygmo/docstrings.cpp

```diff
@@ -1695,6 +1695,64 @@ This method will return the random seed used internally by this uda.
 )";
 }
 
+std::string bee_colony_docstring()
+{
+    return R"(__init__(gen = 1, limit = 1, seed = random)
+
+Artificial Bee Colony
+
+Args:
+    gen (``int``): number of generations
+    limit (``int``): maximum number of trials for abandoning a source (default value is 1)
```

Comment:
remove the default value. This is already in the above constructor. For random it makes a bit more sense as it explains what random is ....